src/Execution/ModelsLoader/PaginatedModelsLoader.php

```diff
@@ -84,7 +85,9 @@ protected function loadRelatedModels(EloquentCollection $parents): EloquentColle
         return $relatedModels->unique(
             // Compare all attributes because there might not be a unique primary key
             // or there could be differing pivot attributes.
```

Comment:
Are pivot attributes also included in `getOriginal()`?

Reply:
Yes, they are:
![image](https://github.com/nuwave/lighthouse/assets/5358638/07e98674-88ff-439a-b7c1-451683d335ef)